pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -158,25 +172,34 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	replicaCount := int32(1)
 	volumeMounts := opspec.CephVolumeMounts(provisionConfig.DataPathMap, false)
 	configVolumeMounts := opspec.RookVolumeMounts(provisionConfig.DataPathMap, false)
+	osdOnPVC := osdProps.pvc.ClaimName != ""
+	osdOnPVCMetadata := osdProps.metadataPVC.ClaimName != ""
+	// When running on PVC, OSDs don't need a bindmount on dataDirHostPath, only the monitors do
```

Comment:
are the crash logs written to it?

Reply:
yes, but it has its own volumes, just like the logs.